recipes-devtools/sbsigntool/sbsigntool/configure-Fixup-build-dependencies-for-cross-compili.patch

```diff
@@ -0,0 +1,53 @@
+From c3533b8da1e1425801d2fc0bcd231e13d593f16b Mon Sep 17 00:00:00 2001
+From: Ricardo Neri <ricardo.neri-calderon@linux.intel.com>
+Date: Tue, 19 Feb 2019 20:07:45 +0800
+Subject: [PATCH] configure: Fixup build dependencies for cross-compiling
+
+When cross-compiling, custom header files and libraries need to be
+specified. sbsign assumes that all the dependencies are located
+under /usr/include and /usr/lib.
+
+Prepend these paths with a placeholder that can be replaced with the
+actual paths once they are resolved.
+
+Upstream status: inappropriate [OE specific]
+
+Signed-off-by: Ricardo Neri <ricardo.neri-calderon@linux.intel.com>
+
+Taken from :
+https://github.com/intel/luv-yocto/tree/master/meta-luv/recipes-devtools/sbsigntool/sbsigntool
+
+Corrected typo error and ported to version 0.9.2
+
+Signed-off-by: Naveen Saini <naveen.kumar.saini@intel.com>
+---
+ configure.ac | 7 +++++--
+ 1 file changed, 5 insertions(+), 2 deletions(-)
+
+diff --git a/configure.ac b/configure.ac
+index 1459e91..3e34c8d 100644
+--- a/configure.ac
++++ b/configure.ac
+@@ -70,7 +70,10 @@ AM_CONDITIONAL(TEST_BINARY_FORMAT, [ test "$EFI_ARCH" = "arm" -o "$EFI_ARCH" = "
+ ##
+ # no consistent view of where gnu-efi should dump the efi stuff, so find it
+ ##
+-for path in /lib /lib64 /usr/lib /usr/lib64 /usr/lib32 /lib/efi /lib64/efi /usr/lib/efi /usr/lib64/efi /usr/lib/gnuefi /usr/lib64/gnuefi ; do
```

Comment:
@Asiderr How about prefixing each path with `$(SYSROOT)` and setting the default SYSROOT value in Makefile `SYSROOT ?= ""` ? Then in recipe you would set the `SYSROOT` to point to the recipe-sysroot dir. Such change could be even potentially accepted to the upstream safeboot repository. Even though the commit description says `Upstream status: inappropriate [OE specific]`.

Reply:
I've replaced that path with the prefix and I've used EXTRA_OECONF to pass the value.

Reply:
Please add to the list for potential contribution. We should at least ask whether it could be accepted. I dont see why not if it does not break the default build when SYSROOT is not set.